add-on/panels/js/panel.jsx

```diff
@@ -340,7 +340,7 @@ loop.panel = (function(_, mozL10n) {
       if (this.props.userProfile && this.props.userProfile.email) {
         return (
           <div className="user-identity">
-            {loop.shared.utils.truncate(this.props.userProfile.email, 24)}
+            {loop.shared.utils.truncate(this.props.userProfile.email, 40)}
```

Comment:
I can't remember if there was one. I *think* it was due to some kind of restraint with the layout (since the value is so specific).
Anyway, I think it's better to try using CSS to fix this.